static/app/utils/discover/eventView.tsx

```diff
@@ -1157,7 +1157,10 @@ class EventView {
     return uniqBy(
       this.getAggregateFields()
         // Only include aggregates that make sense to be graphable (eg. not string or date)
-        .filter((field: Field) => isLegalYAxisType(aggregateOutputType(field.field)))
+        .filter(
+          (field: Field) =>
+            isLegalYAxisType(aggregateOutputType(field.field)) && !isEquation(field.field)
```

Comment:
@wmak would equations ever show up as y-axis options in Dashboards?

Reply:
They will, this is temporary until I add support on the event-stats endpoint for equations, and I'll format it correctly when I add it back, right now this is appearing like this
<img width="449" alt="image" src="https://user-images.githubusercontent.com/4205004/121961503-826bdd80-cd35-11eb-955b-2aaa1e90f80a.png">